eZ/Publish/API/Repository/Tests/LocationServiceTest.php

```diff
@@ -3238,6 +3238,126 @@ public function testMoveSubtreeUpdatesPathIdentificationString(): void
         );
     }
 
+    /**
+     * Test that is_visible is set correct for children when moving a location where a child is hidden by content (not by location).
+     *
+     * @covers \eZ\Publish\API\Repository\LocationService::moveSubtree
+     * depends eZ\Publish\API\Repository\Tests\LocationServiceTest::testLoadLocation
```

Comment:
This line does not do anything in this form and can be removed.

Reply:
Most of the other tests has dependencies to other tests. I assume it is in order to avoid a lot of false positives if some base test starts failing. In the case here I replaced `@depends` with `depends` in order to run the this test alone using filters while developing the test itself. I then forgot to re-add the `@` before commit.
I have anyway now removed the line in a99fb158a9447ae40c7638048cd981a209de12d1